app/gui/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -561,13 +587,18 @@ impl Container {
         let action_bar = &model.action_bar.frp;
         let registry = &model.registry;
         let selection = Animation::new(network);
+        let width_anim = Animation::new(network);
+
+        let on_down = model.view.resize_grip.on_event_capturing::<mouse::Down>();
+        let on_up_source = scene.on_event::<mouse::Up>();
```

Comment:
`source` suggests this is a source node. Why does this need the prefix, and `on_down` does not?